src/Mod/Spreadsheet/App/PropertySheet.cpp

```diff
@@ -124,22 +127,34 @@ bool PropertySheet::isValidAlias(const std::string &candidate)
     if (ExpressionParser::isTokenAUnit(candidate))
         return false;
 
-    /* Check to make sure it doesn't match a cell reference */
-    if (boost::regex_match(candidate.c_str(), cm, gen)) {
-        static const boost::regex e("\\${0,1}([A-Z]{1,2})\\${0,1}([0-9]{1,5})");
-
-        if (boost::regex_match(candidate.c_str(), cm, e)) {
-            const boost::sub_match<const char *> colstr = cm[1];
-            const boost::sub_match<const char *> rowstr = cm[2];
+    App::Expression* ExpressionResult = 0;
```

Comment:
```suggestion
    App::Expression* ExpressionResult = nullptr;
```

Reply:
Ah, thanks. I was not aware that there is a difference since in other places of the code a '0' is used as if that would be the equivalent of a nullptr.

Reply:
nullptr was introduced in c++11, both works but nullptr is the preferred way nowadays in c++ land